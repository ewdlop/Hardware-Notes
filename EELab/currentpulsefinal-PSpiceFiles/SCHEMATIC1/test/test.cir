** Profile: "SCHEMATIC1-test"  [ D:\Cadence\SPB_17.2\tools\bin\av-pspicefiles\schematic1\test.sim ] 

** Creating circuit file "test.cir" 
** WARNING: THIS AUTOMATICALLY GENERATED FILE MAY BE OVERWRITTEN BY SUBSEQUENT SIMULATIONS

*Libraries: 
* Profile Libraries :
* Local Libraries :
* From [PSPICE NETLIST] section of C:\Users\Raymond Lei\cdssetup\OrCAD_PSpice\17.2.0\PSpice.ini file:
.lib "nomd.lib" 

*Analysis directives: 
.OPTIONS ADVCONV
.PROBE64 V(alias(*)) I(alias(*)) W(alias(*)) D(alias(*)) NOISE(alias(*)) 
.INC "..\SCHEMATIC1.net" 


.END
