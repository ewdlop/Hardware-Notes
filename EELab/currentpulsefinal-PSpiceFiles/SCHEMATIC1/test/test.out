
**** 10/10/16 00:18:06 ******* PSpice Lite (March 2016) ******* ID# 10813 ****

 ** Profile: "SCHEMATIC1-test"  [ D:\Cadence\SPB_17.2\tools\bin\av-pspicefiles\schematic1\test.sim ] 


 ****     CIRCUIT DESCRIPTION


******************************************************************************




** Creating circuit file "test.cir" 
** WARNING: THIS AUTOMATICALLY GENERATED FILE MAY BE OVERWRITTEN BY SUBSEQUENT SIMULATIONS

*Libraries: 
* Profile Libraries :
* Local Libraries :
* From [PSPICE NETLIST] section of C:\Users\Raymond Lei\cdssetup\OrCAD_PSpice\17.2.0\PSpice.ini file:
.lib "nomd.lib" 

*Analysis directives: 
.OPTIONS ADVCONV
.PROBE64 V(alias(*)) I(alias(*)) W(alias(*)) D(alias(*)) NOISE(alias(*)) 
.INC "..\SCHEMATIC1.net" 



**** INCLUDING SCHEMATIC1.net ****
* source AV
I_I1         0 N00171 DC 0Adc AC 0Aac
+PULSE 0 10 0 1p 1p 1 2
I_I2         N00171 0 DC 0Adc AC 0Aac
+PULSE 0 10 0.01 1p 1p 1 2
R_R1         N00171 0  1k TC=0,0 

**** RESUMING test.cir ****
.END

**** 10/10/16 00:18:06 ******* PSpice Lite (March 2016) ******* ID# 10813 ****

 ** Profile: "SCHEMATIC1-test"  [ D:\Cadence\SPB_17.2\tools\bin\av-pspicefiles\schematic1\test.sim ] 


 ****     SMALL SIGNAL BIAS SOLUTION       TEMPERATURE =   27.000 DEG C


******************************************************************************



 NODE   VOLTAGE     NODE   VOLTAGE     NODE   VOLTAGE     NODE   VOLTAGE


(N00171)    0.0000 




    VOLTAGE SOURCE CURRENTS
    NAME         CURRENT


    TOTAL POWER DISSIPATION   0.00E+00  WATTS



          JOB CONCLUDED

**** 10/10/16 00:18:06 ******* PSpice Lite (March 2016) ******* ID# 10813 ****

 ** Profile: "SCHEMATIC1-test"  [ D:\Cadence\SPB_17.2\tools\bin\av-pspicefiles\schematic1\test.sim ] 


 ****     JOB STATISTICS SUMMARY


******************************************************************************



  Total job time (using Solver 1)   =        0.00
